src/app/header/header.component.html

```diff
@@ -0,0 +1,15 @@
+<header class="header-container">
+    <div class="header">
+        <!-- Just random logo -->
+        <div class="logo-container" routerLink="/ads">
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ0kJ1egqdu7Z3-HEJZxg6rdtotw3_qktoYzg&usqp=CAU" class="logo">
```

Comment:
use a logo URL in readonly property of the Component file. Much better to move it out the file into a more appropriate file as "constants" or something like that.